### DISCUSSION TITLE
something like navigation map in the main page

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#11436] [General - Components / Directives / etc] [answered]
Dear Quasar team,

Could you please suggest some UI component that looks like a navigation map. So when user login the system, he can click the link in the navigation map and without clicking menu. thanks!

**Comment by MilosPaunovic** [comment, marked as answer]
Whad do you mean by navigation map?

Maybe breadcrumbs could help you:
[https://quasar.dev/vue-components/breadcrumbs](https://quasar.dev/vue-components/breadcrumbs)

**Comment by 549393092** [comment]
@MilosPaunovic Just attached a picture, maybe something like a roadmap, thanks!
![example](https://user-images.githubusercontent.com/38575971/142858560-69cdff7d-da17-4c0e-a051-d307ad88b05e.png)
.
